Documentation/security/policy/language.rst

```diff
@@ -503,6 +504,23 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../../examples/policies/l3/cidr/cidr.json
 
+.. _cidr_select_nodes:
+
+Selecting nodes with CIDR / ipBlock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../beta.rst
+
+By default, CIDR-based selectors do not match in-cluster entities (pods or nodes).
+Optionally, the policy engine can be directed to select nodes by CIDR / ipBlock.
+This requires Cilium to be configured with ``--policy-cidr-match-mode=nodes`` or
+the equivalent Helm value ``policyCIDRMatchMode: nodes``. It is safe to toggle this
+option on a running cluster, and toggling the option affects neither upgrades nor downgrades.
+
+When this is enabled, every agent will allocate a distinct local
+:ref:`security identity <security_identities>` for all other nodes. This will cause
```

Comment:
Minor nit: I see the security identities doc as a developer-focused doc around bitpatterns that is not an API and not user-facing, subject to change without warning. Of course we would only change it if we see a reason and if it's not likely to impact user traffic. I'm a little bit concerned that by cross-linking that doc from this network policy doc which is clearly user-facing and API, we might be giving the impression of stronger guarantees in that other doc than what we intend to.

I think that if we're trying to cross-link here for some specific reason, we should be linking to the user-facing docs like https://docs.cilium.io/en/stable/security/network/identity/#identity-based or similar.